packages/babel-parser/src/parser/expression.js

```diff
@@ -576,13 +586,8 @@ export default class ExpressionParser extends LValParser {
         this.expect(tt.bracketR);
         return this.finishNode(node, "OptionalMemberExpression");
       } else if (this.eat(tt.parenL)) {
-        const possibleAsync = this.atPossibleAsync(base);
-
         node.callee = base;
-        node.arguments = this.parseCallExpressionArguments(
-          tt.parenR,
-          possibleAsync,
-        );
```

Comment:
I removed this, as I don't see a way what an `optionalCallExpression` has anything todo with the `async` keyword. No tests failed or changed. I guess this was copied wrong.